config/install/filter.format.full_html.yml

```diff
@@ -31,3 +32,19 @@ filters:
     status: true
     weight: 11
     settings: {  }
+  filter_html:
```

Comment:
Filter html on full_html that sounds wrong to me?

Reply:
Right, that shouldn't be there